docs/data/material/pages.ts

```diff
@@ -106,8 +106,8 @@ const pages: MuiPage[] = [
         children: [
           { pathname: '/material-ui/react-box' },
           { pathname: '/material-ui/react-container' },
-          { pathname: '/material-ui/react-grid' },
-          { pathname: '/material-ui/react-grid2', title: 'Grid v2', newFeature: true },
+          { pathname: '/material-ui/react-grid', legacy: true },
```

Comment:
I'd be better to have `deprecated` flag. I think legacy has a slightly different meaning because it won't be removed but deprecated API will.

Reply:
Done 👌🏼